frontends/concrete-python/examples/levenshtein_distance/IBAN_name_check.ipynb

```diff
@@ -0,0 +1,363 @@
+    "\n",
+    "In this small tutorial, we show how to use our TFHE Levenshtein distance computations to perform such a privacy-preserving check, very simply and directly in Python. This tutorial can be easily configured, to change for example the way strings are normalized before encryption and comparison. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc96a80f-0b14-4e64-a33f-31a60351453d",
+   "metadata": {},
+   "source": [
+    "## Importing our FHE Levenshtein computations\n",
+    "\n",
+    "One can have a look to this file to see how the FHE computations are handled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "56ba9e20-ca46-4aa6-a0f7-86ca13480a52",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from levenshtein_distance import *\n",
+    "from time import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "494cd58c-ea28-4547-92c5-80ed4ba83964",
+   "metadata": {},
+   "source": [
+    "## Define the comparison functions\n",
+    "\n",
+    "FHE computation will happen in `calculate_and_return`, if `fhe_or_simulate` is set to `fhe`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2410b6b3-0c21-4178-b8dc-f734ec0afd40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Arguments:\n",
+    "    \"\"\"Fake an argument parser, to be able to reuse the functions from levenshtein_distance.py.\"\"\"\n",
+    "    def __init__(self, max_string_length, show_mlir = False, show_optimizer = False):\n",
+    "        self.max_string_length = max_string_length\n",
+    "        self.show_mlir = show_mlir\n",
+    "        self.show_optimizer = show_optimizer\n",
+    "\n",
+    "def normalized_string(st):\n",
+    "    \"\"\"Normalize a string, to later make that the distance between non-normalized\n",
+    "    string 'John Doe' and 'doe john' is small. This function can be configured depending\n",
+    "    on the needs.\n",
+    "    \"\"\"\n",
+    "\n",
+    "    # Force lower case\n",
+    "    st = st.lower()\n",
+    "\n",
+    "    # Replace - and . by spaces\n",
+    "    st = st.replace('-', ' ')\n",
+    "    st = st.replace('.', ' ')\n",
+    "    \n",
+    "    # Sort the words and join\n",
+    "    words = st.split()\n",
+    "    st = \"\".join(sorted(words))\n",
+    "\n",
+    "    return st\n",
+    "\n",
+    "def compare_IBAN_names(string0: str, string1: str, fhe_or_simulate: str):\n",
+    "    \"\"\"Compare two IBAN names: first, normalize the strings, then compute in FHE (look in \n",
+    "    calculate_and_return for FHE details).\"\"\"\n",
+    "    # Normalize strings\n",
+    "    string0 = normalized_string(string0)\n",
+    "    string1 = normalized_string(string1)\n",
+    "    args = Arguments(max_string_length = max(len(string0), len(string1)))\n",
+    "\n",
+    "    alphabet = Alphabet.init_by_name(\"name\")\n",
```

Comment:
Why not setting the alphabet name to `IBAN`?
```python
alphabet = Alphabet.init_by_name("IBAN")
```

Reply:
I would say: this alphabet can be used for whatever "names" operations, not only for IBAN

Reply:
Yes, but this example is for IBANs, `iban` would be more descriptive than `name`.

Reply:
per https://en.wikipedia.org/wiki/International_Bank_Account_Number, I think IBAN is more the number than the name associated with it. I would really think "name" is clear enough here.

Reply:
It's the `name` of the alphabet. The alphabet named `iban` would contain every character that can appear in an IBAN number. I think it makes sense, and it's a lot more sane than an alphabet named `name`. But I won't push further, as you wish.

Reply:
but in an IBAN, there could be numbers, we don't want that here, we want to check account names associated with the numbers. And what we do could be used for whatever name comparison, so I would prefer to keep something general.  

Let's stop this conversation, it's not a big deal!